### PR TITLE
Fix Cronet end2end tests

### PR DESCRIPTION
**Pull request by muxi**
Add additional read request to Cronet in order to trigger on_succeeded() callback from Cronet.

### CODE REVIEW

File: src/core/ext/transport/cronet/transport/cronet_transport.c [hunk 1]
```diff
@@ -58,7 +58,7 @@
   } while (0)
 
 /* TODO (makdharma): Hook up into the wider tracing mechanism */
-int grpc_cronet_trace = 0;
+int grpc_cronet_trace = 1;
```
Comment (makdharma) [inline src/core/ext/transport/cronet/transport/cronet_transport.c]: Set it to 0 to reduce log noise?

**Comment by muxi** [comment]
@makdharma Fixed the end2end tests with new core.

Craig said `close_error` is phasing out so right now I am marking that particular test requiring it (max_message_length) as `NOT SUPPORTED`.

**Comment by muxi** [comment]
@makdharma Fixed the end2end tests with new core.

Craig said `close_error` is phasing out so right now I am marking that particular test requiring it (max_message_length) as `NOT SUPPORTED`. 

Request another review for the recent commits.

**Comment by muxi** [comment]
test this please

**Comment by muxi** [comment]
test this please

**Comment by muxi** [comment]
Tests are failing with issues:
Basic Tests Windows - #8996
gRPC performance pull requests - #8997

**Pull request merged**